core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/ModelEvaluation.kt

```diff
@@ -0,0 +1,10 @@
+package org.evomaster.core.problem.rest.classifier
+
+data class ModelEvaluation(
+    val accuracy: Double,
+    val precision400: Double,
+    val recall400: Double,
+    val f1Score400: Double,
+    val mcc: Double
+)
```

Comment:
should have an `init` block to validate the inputs, ie, each should be in [0,1], or throw IllegalArgumentException otherwise